app/io/pathfinder/models/Commodity.scala

```diff
@@ -5,11 +5,12 @@ import javax.persistence.{Column, Entity, GeneratedValue, GenerationType, Id, Ma
 import com.avaje.ebean.Model
 import com.fasterxml.jackson.annotation.JsonIgnore
 import io.pathfinder.data.{Resource, EbeanCrudDao}
+import io.pathfinder.websockets.WebSocketDao
 import play.api.libs.json.{Json, Format}
 
 object Commodity {
     val finder: Model.Find[Long,Commodity] = new Model.Finder[Long,Commodity](classOf[Commodity])
-    object Dao extends EbeanCrudDao[Long,Commodity](finder)
+    object Dao extends WebSocketDao[Commodity](finder)
```

Comment:
Why is this dao specific to web sockets? I don't think that the data layer should be coupled to the transport layer.